src/datasets/dto/update-dataset.dto.ts

```diff
@@ -291,13 +291,14 @@ export class UpdateDatasetDto extends OwnableDto {
   readonly dataQualityMetrics?: number;
 
   @ApiProperty({
-    type: String,
-    required: true,
+    type: [String],
+    required: false,
     description:
       "First name and last name of principal investigator(s). If multiple PIs are present, use a semicolon separated list. This field is required if the dataset is a Raw dataset.",
   })
+  @IsOptional()
```

Comment:
The description might also need to be changed?

Something like `Multiple PIs can be provided as separate strings in the array`?